src/Playwright/API/Generated/IAPIRequestContext.cs

```diff
@@ -0,0 +1,185 @@
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Microsoft.Playwright
+{
+    /// <summary>
+    /// <para>
+    /// This API is used for the Web API testing. You can use it to trigger API endpoints,
+    /// configure micro-services, prepare environment or the service to your e2e test.
+    /// </para>
+    /// <para>
+    /// Each Playwright browser context has associated with it <see cref="IAPIRequestContext"/>
+    /// instance which shares cookie storage with the browser context and can be accessed
+    /// via <see cref="IBrowserContext.APIRequest"/> or <see cref="IPage.APIRequest"/>.
+    /// It is also possible to create a new APIRequestContext instance manually by calling
+    /// <see cref="IAPIRequest.NewContextAsync"/>.
+    /// </para>
+    /// <para>**Cookie management**</para>
+    /// <para>
+    /// <see cref="IAPIRequestContext"/> retuned by <see cref="IBrowserContext.APIRequest"/>
+    /// and <see cref="IPage.APIRequest"/> shares cookie storage with the corresponding
+    /// <see cref="IBrowserContext"/>. Each API request will have <c>Cookie</c> header populated
+    /// with the values from the browser context. If the API response contains <c>Set-Cookie</c>
+    /// header it will automatically update <see cref="IBrowserContext"/> cookies and requests
+    /// made from the page will pick them up. This means that if you log in using this API,
+    /// your e2e test will be logged in and vice versa.
+    /// </para>
+    /// <para>
+    /// If you want API requests to not interfere with the browser cookies you shoud create
+    /// a new <see cref="IAPIRequestContext"/> by calling <see cref="IAPIRequest.NewContextAsync"/>.
+    /// Such <c>APIRequestContext</c> object will have its own isolated cookie storage.
+    /// </para>
+    /// </summary>
+    public partial interface IAPIRequestContext
+    {
+        /// <summary>
+        /// <para>
+        /// Creates a new <see cref="IFormData"/> instance which is used for providing form
+        /// and multipart data when making HTTP requests.
+        /// </para>
+        /// </summary>
+        IFormData CreateFormData();
```

Comment:
Would it make sense to make it static?

Reply:
We don't have any other static methods, so I'd vote for no for now.